client/download.go

```diff
@@ -66,7 +66,7 @@ func (client *StorClient) downloadWorker(id int, shasForDownload <-chan hashutil
 				"sha256": sha.String(),
 			}).Debugf("File %s exists - skip download", filepath)
 
-			downloadedFilesStat <- DownStat{skip: true}
+			downloadedFilesStat <- DownStat{Status: DOWN_SKIP}
```

Comment:
I would also prefer to have the `DOWN_FAIL` values explicitly assigned on line 102.

Reply:
`DOWN_FAIL` is default, but I understand - explicitly is more readable... I will do it..